UI/Web/src/app/_services/gestures.service.ts

```diff
@@ -0,0 +1,23 @@
+let swipeCoord = [0, 0];
+let swipeTime = new Date().getTime();
+
+export function swipe(e: TouchEvent, when: string): string | undefined {
+
+    const coord: [number, number] = [e.changedTouches[0].clientX, e.changedTouches[0].clientY];
+    const time = new Date().getTime();
+
+    if (when === 'start') {
+        swipeCoord = coord;
+        swipeTime = time;
+    } else if (when === 'end') {
+        const direction = [coord[0] - swipeCoord[0], coord[1] - swipeCoord[1]];
+        const duration = time - swipeTime;
+
+        if (duration < 1000 //
```

Comment:
Does this swiping handle up/down to handle the up/down mode of the manga reader?

Reply:
Right now it strictly handles horizontal left/right (prev/next?).
What is the up/down (top/bottom?) mode and how does it work?

Reply:
Click the reading mode button once on the menu and you'll see. We need to ensure this doesn't run when we are in webtoon mode as well.

Reply:
Right now it's tested working with up/down reading mode, but still using the horizontal axis.
I believe we can extend the coordinate logic to switch to vertical when on this reading mode.

Webtoon scrolling and functionality doesn't seem to be affected by this, even though page numbers do update in the slider.
I'll add a check for webtoon mode before enabling the swipe gesture.

Reply:
I think it would be better to return early on the handler if webtoon mode is active.

Reply:
You replied in a comment rather than on this thread, but the reason why doing it in the handler is that the code is much cleaner than loading up the template. You can already see the complexity that is added due to the template being overloaded with complex logic.